base/exports.jl

```diff
@@ -1148,7 +1148,6 @@ export
     ismarked,
     isopen,
     isreadonly,
-    limit_output,
```

Comment:
Just before new year's, it was part of #13825 - so I think I agree, the number of packages that develop strictly against 0.5-dev has been pretty low and if they're relying on things that are only on master they're probably used to having to keep up anyway.